src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let properties = sourceString.split(';');
+
+  properties = properties.slice(0, properties.length - 1);
+
+  properties = properties
+    .map(property => property.trim())
+    .filter(property => property.length !== 0);
+
+  properties = properties.map(property => {
+    return property.split(':');
+  });
```

Comment:
Array methods `slice()`, `map()`, `filter()` return new array. You can chain this functions instead of reassigning `properties` multiple times.